monsteer/plugin/spikeReport.cpp

```diff
@@ -49,19 +50,26 @@ namespace
 const uint32_t INTERNAL_TIMEOUT = 500;
 }
 
+URI toHostAndPort( const URI& uri )
+{
+    URI out;
+    out.setHost( uri.getHost( ));
+    out.setPort( uri.getPort( ));
+    return out;
+}
+
 SpikeReport::SpikeReport( const brion::SpikeReportInitData& pluginData )
-    : _uri( pluginData.getURI( ))
+    : _uri( toHostAndPort( pluginData.getURI( )))
```

Comment:
Yes, the problem is that it is not possible to construct a correct servus::URI from a string without a schema, otherwise it is parsed as a path which creates problems down the line. I was in the process of adding a static factory method to Servus for that purpose but postponed it once again. Maybe I should really do that.